nfs/src/rpc_task.cpp

```diff
@@ -546,18 +547,25 @@ static void write_callback(
             mb->clear_flushing();
         }
     } else {
-        /*
-         * Since the api failed and can no longer be retried, set write_error
-         * and do not clear dirty flag.
-         */
-        AZLogError("[{}] Write [{}, {}) failed with status {}: {}",
-                   ctx->get_ino(),
-                   mb->offset + ctx->get_count(), mb->offset + mb->length,
-                   status, errstr);
-
-        assert(mb->is_dirty());
-        mb->clear_flushing();
-        inode->set_write_error(status);
+        if (NFS_STATUS(res) == NFS3ERR_JUKEBOX) {
+            AZLogDebug("[{}] JUKEBOX error write, off: {}, len: {}",
+                       ctx->get_ino(), mb->offset, mb->length);
```

Comment:
mb->offset and mb->length are not very useful as they define the complete write. This write is
mb->offset+bc->pvt and mb->length-bc->pvt.